### PR TITLE
update op_add and op_remove function for support round_robin load_balance

### PR DESCRIPTION
**Pull request by wangfakang**
@cubicdaiya  update api op_add and op_remove and make support  round_robin.
Please code review, Thank you.

### CODE REVIEW

**Comment by cubicdaiya** [comment]
Thank you for your PR. I'll review in the near future.

File: src/ngx_dynamic_upstream_op.c [hunk 1]
```diff
@@ -371,7 +380,8 @@ ngx_dynamic_upstream_op_remove(ngx_http_request_t *r, ngx_dynamic_upstream_op_t
                       __LINE__);
         return NGX_ERROR;
     }
-
+
```
Comment (cubicdaiya) [inline src/ngx_dynamic_upstream_op.c]: Remove spaces on the empty-line, please.

**Comment by wangfakang** [comment]
@cubicdaiya 
oh, I'm sorry, I have updated the code style.

**Comment by cubicdaiya** [comment]
Hi, @wangfakang

Sorry for late. Merged your PR. Thanks.

**Pull request merged**